### PR TITLE
SEO-176062-Xamarin UG-page issue links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/DataForm/Editing.md [hunk 1]
```diff
@@ -719,8 +719,8 @@ You can download the entire source code of this demo from here [Two-wayDataBindi
...
+[How to register common editor for same data type properties in Xamarin.Forms DataForm business object](https://support.syncfusion.com/kb/article/8921/how-to-register-common-editor-for-same-data-type-properties-in-xamarin-forms-dataform?isInternalRefresh=False)                                                                                                                                                                                   
+[How to change the editor visibility based on another editor in Xamarin.Forms DataForm (SfDataForm)](https://support.syncfusion.com/kb/article/9748/how-to-change-the-editor-visibility-based-on-another-editor-in-xamarin-forms-dataform?isInternalRefresh=False)                                                                                                                                                                  
+[How to update editor value based on another editor in Xamarin.Forms DataForm (SfDataForm)](https://support.syncfusion.com/kb/article/9745/how-to-update-editor-value-based-on-another-editor-in-xamarin-forms-dataform-sfdataform?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/DataForm/Editing.md:726]: Check KB links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/DataForm/Editing.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/DataForm/Editing.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**